### PR TITLE
Jobradovic done test store

### PR DESCRIPTION
**Pull request by jolpazy**

### CODE REVIEW

File: web/src/main/frontend/src/components/MultipleChoiceQuestion.js [hunk 1]
```diff
@@ -5,31 +5,28 @@ import Checkbox from './ui/Checkbox';
 @observer
 class MultipleChoiceQuestion extends Component {
   render() {
+    let options = this.props.options.map(option => (
+      <div className="row row-eq-height answer" key={option.id}>
+        <div className="col-2">
+          <Checkbox
+            checked={
+              this.props.solution && this.props.solution === option.id
```
Comment (iscekic) [inline web/src/main/frontend/src/components/MultipleChoiceQuestion.js]: `this.props.solution === option.id` should be enough.

File: web/src/main/frontend/src/components/ReviewTest.js [hunk 2]
```diff
@@ -1,10 +1,50 @@
...
 
+@inject('completedTest')
+@observer
 class ReviewTest extends Component {
+  questionsToComponents = (question, index) => {
+    switch (question.type) {
+      case 'CodeChallenge':
+        return (
+          <CodeChallengeQuestion
+            index={index + 1}
```
Comment (iscekic) [inline web/src/main/frontend/src/components/ReviewTest.js]: Why is this here?

File: web/src/main/frontend/src/components/MultipleChoiceQuestion.js (hunk 1 above)
Comment (jolpazy) [reply web/src/main/frontend/src/components/MultipleChoiceQuestion.js]: still need to return null and not false in case there is no solution passed, since false on checked property makes it read only, and returning null means there's no checked property at all

File: web/src/main/frontend/src/components/ReviewTest.js (hunk 2 above)
Comment (jolpazy) [reply web/src/main/frontend/src/components/ReviewTest.js]: So the first question would be 1. not 0. in the UI

File: web/src/main/frontend/src/components/MultipleChoiceQuestion.js (hunk 1 above)
Comment (jolpazy) [reply web/src/main/frontend/src/components/MultipleChoiceQuestion.js]: Idea is that if we pass solution to MultipleChoice question, it makes it's checkboxes readonly with checked and disabled properties, if not, it is normal checkbox

File: web/src/main/frontend/src/components/MultipleChoiceQuestion.js [hunk 3]
```diff
@@ -5,31 +5,24 @@ import Checkbox from './ui/Checkbox';
 @observer
 class MultipleChoiceQuestion extends Component {
   render() {
+    let options = this.props.options.map(option => (
+      <div className="row row-eq-height answer" key={option.id}>
+        <div className="col-2">
+          <Checkbox
+            checked={this.props.solution === option.id ? true : null}
```
Comment (iscekic) [inline web/src/main/frontend/src/components/MultipleChoiceQuestion.js]: `this.props.solution === option.id || null` and we can merge. :smile:
